e2e/tests/core/04-channel/channel_test.go

```diff
@@ -43,8 +54,49 @@ func (s *ChannelUpgradeTestSuite) TestChannelUpgrade() {
 		s.Require().NoError(err)
 		s.AssertValidTxResponse(txResp)
 
+		s.Require().NoError(testsuite.UnmarshalMsgResponses(txResp, &msgChanUpgradeInitRes))
+
 		channel, err := s.QueryChannel(ctx, chainA, channelA.PortID, channelA.ChannelID)
 		s.Require().NoError(err)
 		s.Require().Equal(channeltypes.INITUPGRADE, channel.State)
 	})
+
+	t.Run("channel upgrade try", func(t *testing.T) {
+		//chainBChannels, err := relayer.GetChannels(ctx, s.GetRelayerExecReporter(), chainB.Config().ChainID)
+		//s.Require().NoError(err)
+		//s.Require().Len(chainBChannels, 1)
+		//
+		//chainBChannel := chainBChannels[len(chainBChannels)-1]
+		//upgradeTimeout := channeltypes.NewUpgradeTimeout(clienttypes.NewHeight(0, 10000), 0)
+		//upgradeFields := channeltypes.NewUpgradeFields(channeltypes.UNORDERED, chainBChannel.ConnectionHops, upgradeVersion)
+		//
+		//// TODO: get channel proof
+		//var channelProof []byte
+		//// TODO: get upgrade proof
+		//var upgradeProof []byte
+		//
+		//msgChannelUpgradeTry := channeltypes.NewMsgChannelUpgradeTry(
+		//	channelA.Counterparty.PortID,
+		//	channelA.Counterparty.ChannelID,
+		//	upgradeFields,
+		//	upgradeTimeout,
+		//	msgChanUpgradeInitRes.Upgrade,
+		//	msgChanUpgradeInitRes.UpgradeSequence,
+		//	channelProof,
+		//	upgradeProof,
+		//	clienttypes.ZeroHeight(), // proof height
+		//	rlyWallet.FormattedAddress(),
+		//)
+		//
+		//txResp, err := s.BroadcastMessages(ctx, chainB, rlyWallet, msgChannelUpgradeTry)
+		//s.Require().NoError(err)
+		//s.AssertValidTxResponse(txResp)
+		//
+		//s.Require().NoError(testsuite.UnmarshalMsgResponses(txResp, &msgChanUpgradeTryRes))
+		//s.Require().True(msgChanUpgradeTryRes.Success)
+		//
+		//channel, err := s.QueryChannel(ctx, chainB, chainBChannel.PortID, chainBChannel.ChannelID)
+		//s.Require().NoError(err)
+		//s.Require().Equal(channeltypes.TRYUPGRADE, channel.State)
+	})
```

Comment:
we can either just remove this, or leave it commented out, I don't have a strong preference, this is a WIP e2e anyway

Reply:
let's maybe leave it actually so it is there as a reference (squashing the commit will remove this completely from git history if it is removed here)